pyproject.toml

```diff
@@ -61,7 +61,7 @@ pytest-recording = "^0.12.1"
 skip-string-normalization = true
 
 [tool.ruff]
-line-length = 100
+line-length = 140
```

Comment:
This feels a bit arbitrary (sure, most line length limits are) but curious if there's a particular reason for 140.

Reply:
140 was the sweet spot for the changes we were trying to push and it still fits in nicely over almost all modern day desktop screens.  Let us know.

Reply:
I think the preference would be to keep the existing size format, black will automatically format the code to stay within the line limit.

Reply:
I reverted the line-length to 100 and then ran black tools/c7n_oci  but it did not fix the errors. Let me know if I am doing something incorrect or if the only way is to manually fix the files

Reply:
black with 'review' mode has the support for formatting the line length.